cmd/kubelet/app/server.go

```diff
@@ -135,11 +135,9 @@ func UnsecuredDependencies(s *options.KubeletServer) (*kubelet.Dependencies, err
 		return nil, err
 	}
 
-	mounter := mount.New(s.ExperimentalMounterPath)
+	mounter := mount.New(s.ExperimentalMounterPath, s.Containerized)
```

Comment:
Please revert this and keep the previous behavior with `mount.NewNsenterMounter` after `s.Containersized` check and with no new argument to `mount.New()`.

If you want to improve this part, then check `s.Containersized` and create "normal" / nsenter mounter and StdWriter / NsenterWriter in `then` /  `else` branches so it gets initialized only once.

Reply:
> You will save yourselves all the New() changes all around the code.

Good advice. Will revert to drop this commit.

> with no new argument to `mount.New()`

`Kubelet` provides a flag `experimental-mounter-path` to support customized mount binary. We'd better keep as it is, right?

Reply:
Yes please